examples/tutorial/00_Setup.ipynb

```diff
@@ -63,8 +65,32 @@
     "    ```\n",
     "    Note: You can replace \"lab\" with \"notebook\" if you prefer the original Jupyter notebook interface.\n",
     "\n",
-    "5. **Test HoloViz imports**:\n",
-    "After launching Jupyter, navigate to this notebook (`tutorial/00_Setup.ipynb`) and run the following cell. This command is a quick check that your environment is set up correctly. You should see the HoloViews, Bokeh, and Matplotlib logos along with a success message.\n"
+    "6. **Test HoloViz imports**:\n",
+    "After launching Jupyter, navigate to this notebook (`tutorial/00_Setup.ipynb`) and run the following cell. This command is a quick check that your environment is set up correctly. You should see the HoloViews, Bokeh, and Matplotlib logos along with a success message.\n",
+    "\n",
+    "--- Other Path\n",
+    "\n",
+    "3. **Create and activate a new conda environment**: You can also use plain conda to install.\n",
+    "    ```bash\n",
+    "    >> conda env create -f env.yml\n",
+    "    >> conda activate holoviz-tutorial\n",
+    "    ```\n",
+    "4. **Download datasets**: We will be using the following datasets; make sure you're in `examples/` directory before running this.\n",
+    "   ```bash\n",
+    "    mkdir data/raster\n",
```

Comment:
On Github at least this section is a bit weirdly formatted 
<img width="843" alt="image" src="https://github.com/holoviz/holoviz/assets/35924738/a443818e-29bb-42ab-bda2-7ef2b795511f">